Readme.md

```diff
@@ -8,7 +8,7 @@ ETH Zurich
 
 Contact  : Christian Gehring [gehrinch ( at ) ethz.ch]
 
-Author(s): C. Dario Bellicoso, Michael Bloesch, Remo Diethelm, Peter Fankhauser, Paul Furgale, Christian Gehring, Hannes Sommer
+Author(s): C. Dario Bellicoso, Michael Bloesch, Remo Diethelm, Peter Fankhauser, Paul Furgale, Christian Gehring, Hannes Sommer, Michael Neunert, Google LLC
```

Comment:
Two things here:
* The authors were apparently in last name alphabetical order before
* What is Google doing here? The other authors don't have affiliations listed so far. Or is it supposed to be an author on its own? Are you sure the company actually contributed? (Even if it an employee of Google contributed that doesn't really imply the companies contribution IMO, unless it specifically paid for the contribution)

Reply:
I do not mind the order and how you handle authors / contributors / etc. in general. I added my name because of previous contributions (https://github.com/ethz-asl/kindr/pull/83) but they were minor, so I do not care so much about it. Google LLC should be listed since this pull request is a contribution by me as a Google employee as part of my work. Feel free to ping me directly if you want to discuss this.

Reply:
@gehrinch , could you decide on that? 
I would strongly oppose putting a company as an separate author. 
What about `Michael Neunert (Google LLC)` instead? The way it is now looks like Google being a separate author, as the only non-human entity. This would be too much of (almost) free advertisement for my taste.